tests/expected/object-bits/insufficient/main.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Checks for error message with an --object-bits value that is too small
+
+// kani-flags: --object-bits 5 --default-unwind 30
```

Comment:
enable-unstable?

Reply:
Can you use --cbmc-args instead? I think I'm just going to nuke --object-bits.

Reply:
Yes, will push this change once you merge #1109.

Reply:
Thanks

Reply:
Done.